docs/content/preview/yugabyte-platform/security/create-kms-config/azure-kms.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Create a KMS configuration using Azure
+headerTitle: Create a KMS configuration using Azure
+linkTitle: Create a KMS configuration
+description: Use YugabyteDB Anywhere to create a KMS configuration for Azure KMS.
+aliases:
+  - /preview/yugabyte-platform/security/create-kms-config
+menu:
+  preview_yugabyte-platform:
+    parent: security
+    identifier: create-kms-config-1-azure-kms
+    weight: 27
+type: docs
+---
+
+<ul class="nav nav-tabs-alt nav-tabs-yb">
+  <li >
+    <a href="{{< relref "./aws-kms.md" >}}" class="nav-link">
+      <i class="fab fa-aws" aria-hidden="true"></i>
+      AWS KMS
+    </a>
+  </li>
+  <li >
+    <a href="{{< relref "./google-kms.md" >}}" class="nav-link">
+      <i class="icon-google" aria-hidden="true"></i>
+      Google KMS
+    </a>
+  </li>  
+
+<li >
+    <a href="{{< relref "./azure-kms.md" >}}" class="nav-link active">
+      <i class="icon-azure" aria-hidden="true"></i>
+      &nbsp;&nbsp;Azure KMS
+    </a>
+  </li>
+
+  <li >
+    <a href="{{< relref "./hashicorp-kms.md" >}}" class="nav-link">
+      <i class="icon-postgres" aria-hidden="true"></i>
+      HashiCorp Vault
+    </a>
+  </li>
+
+</ul>
+
+Encryption at rest uses universe keys to encrypt and decrypt universe data keys. You can use the YugabyteDB Anywhere UI to create key management service (KMS) configurations for generating the required universe keys for one or more YugabyteDB universes. Encryption at rest in YugabyteDB Anywhere supports the use of Microsoft Azure KMS. 
+
+Conceptually, Azure KMS consists of a key vault containing one or more keys, with each key capable of having multiple versions.
+
+Before defining a KMS configuration with YugabyteDB Anywhere, you need to create a key vault through the [Azure portal](https://docs.microsoft.com/en-us/azure/key-vault/general/quick-create-portal). The following settings are required:
+
+- Set the vault permission model as Vault access policy.
+- Add the application to the key vault access policies with the minimum key management operations permissions of Get, List, Create, and cryptographic operations permissions of Unwrap Key, Wrap Key. 
```

Comment:
Can we also change this to reflect the new changes in [this PR](https://github.com/yugabyte/yugabyte-db/pull/13916).

Reply:
@Sahith02 I'm not sure what this comment is referring to.

Reply:
@lizayugabyte I mean changing line 53 to what is modified in the [comment here](https://github.com/yugabyte/yugabyte-db/pull/13916#discussion_r976085912).